.cirrus.yml

```diff
@@ -508,11 +485,13 @@ success_task:
 
     depends_on:  # ignores any dependent task conditions
         - "gating"
-        - "build_each_commit"
         - "vendor"
         - "varlink_api"
+        - "build_each_commit"
+        - "meta"
         - "testing"
-        - "special_testing"
+        - "special_testing_rootless"
+        - "special_testing_in_podman"
         - "optional_testing"
```

Comment:
Didn't you just remove this above?

Reply:
Oh, well, I guess you'll have to open up a PR to fix ***MY*** mess then :stuck_out_tongue:

Reply:
okay, fixed and pushed.  thx :smile: